.github/workflows/haskell.yml

```diff
@@ -61,6 +62,12 @@ jobs:
           - 8.4.4+debug
         cabal_version: ["3.4"]
         include:
+          - name: ghc_HEAD
+            ghc_version: 9.2.0.20210422 
+            build: cabal-v2
+            cabal_build_options: "--allow-newer=hsc2hs"
```

Comment:
Use `cabal_build_options: "--allow-newer"` to allow newer versions of everything.